portfolio/src/main/webapp/CommentHandler.js

```diff
@@ -0,0 +1,31 @@
+export default class CommentHandler {
+  /**
+   * Loads all comments from backend.
+   * This function is called when the page is loaded.
+   * @param {HTML div object} commentBox The div in document with id "comment-box"
+   * @param {HTML template} commentCell The template for one comment
+   * @private
+   */
+  async loadComments_(commentBox, commentCell) {
+    let commentResponse = undefined;
+    try {
+      commentResponse = await fetch('/comment');
+    } catch (e) {
+      throw e;
+      return;
```

Comment:
Your code will never reach this line because you threw an exception. You actually don't need 2 separate try/catch blocks, right? There's no code specific to either block in the catch.

Reply:
Yeah you're right. We don't need to set alert for different cases here.